metricbeat/module/system/filesystem/_meta/docs.asciidoc

```diff
@@ -12,20 +12,26 @@ This metricset is available on:
 [float]
 === Configuration
 
-*`filesystem.ignore_types`* - A list of filesystem types to ignore. Metrics will
+*`filesystem.ignore_types`* - An array of filesystem types to ignore. Metrics will
 not be collected from filesystems matching these types. This setting also
```

Comment:
>This setting also affects the `fsstats` metricset

Does it _only_ affect that metricset? The "also" could imply that it affects some un-named metricset _as well ass_ the `fsstats` metricset.

Reply:
Good point. I've removed the "also" and moved this sentence to the end of the paragraph. Changes in [`0c7880b` (#36946)](https://github.com/elastic/beats/pull/36946/commits/0c7880b04567f0bf28d37accdbd9547ecc767d38)